src/test/testConstants.ts

```diff
@@ -25,11 +25,69 @@ export const NoOpLibrary: Library.ILibrary = {
     libraryDefinitions: new Map(),
 };
 
+export const CreateFooAndBarRecordDefinedFunction: PQP.Language.Type.DefinedFunction = PQP.Language.TypeUtils.createDefinedFunction(
+    false,
+    [],
+    PQP.Language.TypeUtils.createDefinedRecord(
```

Comment:
FYI - I found out that policheck will flag "foo" and "foobar", so I've started avoiding them.

Reply:
> FYI - I found out that policheck will flag "foo" and "foobar", so I've started avoiding them.

That's like saying you can't use the variable `x` in math. I'll keep it in mind for the future, and plan on handling any policheck fallout if it shows up later.